cmd/crypto/vault.go

```diff
@@ -51,11 +51,12 @@ type VaultAppRole struct {
 
 // VaultConfig represents vault configuration.
 type VaultConfig struct {
-	Endpoint  string    `json:"endpoint"` // The vault API endpoint as URL
-	CAPath    string    `json:"-"`        // The path to PEM-encoded certificate files used for mTLS. Currently not used in config file.
-	Auth      VaultAuth `json:"auth"`     // The vault authentication configuration
-	Key       VaultKey  `json:"key-id"`   // The named key used for key-generation / decryption.
-	Namespace string    `json:"-"`        // The vault namespace of enterprise vault instances
+	Endpoint     string    `json:"endpoint"` // The vault API endpoint as URL
+	CAPath       string    `json:"-"`        // The path to PEM-encoded certificate files used for mTLS. Currently not used in config file.
+	Auth         VaultAuth `json:"auth"`     // The vault authentication configuration
+	Key          VaultKey  `json:"key-id"`   // The named key used for key-generation / decryption.
+	Namespace    string    `json:"-"`        // The vault namespace of enterprise vault instances
+	KeyStorePath string    `json:"-"`
```

Comment:
a comment for `KeyStorePath` would be useful for other contributors in the future